demos/drawer/permanent-drawer-above-toolbar.html

```diff
@@ -51,14 +52,18 @@
       }
 
       /* Place drawer above toolbar shadow. */
-      .mdc-permanent-drawer {
+      .mdc-drawer--permanent {
         z-index: 2;
       }
 
       .demo-main {
         padding-left: 16px;
       }
 
+      .mdc-demo__control {
```

Comment:
Let's rename this to `demo-form-field`.

`mdc-form-field` is a special RTL-aware class that wraps a checkbox, radio, or switch, plus its adjacent `<label>`.

And our new convention is to prefix CSS class names for demo pages with `demo-`.

_(I generally tend to just `s/^mdc-/demo-/` the class name I'm customizing for the demo 🙂 )_

Reply:
gotcha makes sense